src/test/java/software/amazon/awssdk/crt/test/ProcessTest.java

```diff
@@ -43,7 +43,7 @@ public void testSetSoftIOHandleLimits() {
             Process.setMaxIOHandlesSoftLimit(softLimit - 1);
         } catch (CrtRuntimeException ex) {
             // make sure it's the not-implemented exception if it was thrown.
-            assertEquals(36, ex.errorCode);
+            assertEquals("AWS_ERROR_UNIMPLEMENTED", ex.errorName);
```

Comment:
Can we build an enum class for these AWS_ERROR_* ?

Reply:
Java CRT exception and error codes needs a major overhaul that's probably out of scope here.

Reply:
Yes, but not in this PR

We should have a definitive list of errors in code, maybe as enums, maybe as a concrete class types per C error code. This definitive list should be code-generated and not hand-typed, because it's not uncommon for new codes to appear. This is a task in our backlog...